source/bdDetect.py

```diff
@@ -655,3 +686,7 @@ def addDeviceScanner(
 		scanForDevices.register(scanFunc)
 		if moveToStart:
 			scanForDevices.moveToEnd(scanFunc, last=False)
+
+
+class FallbackDevicesStore:
+    fallBackDevices: List[Tuple[str, DeviceType, str]] = []

```

Comment:
I think I prefer this to become a set:
```suggestion
    fallBackDevices: set[Tuple[str, DeviceType, str]] = set()
```